src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -21,13 +21,15 @@ internal abstract class SynthesizedParameterSymbolBase : ParameterSymbol
         private readonly string _name;
         private readonly RefKind _refKind;
         private readonly DeclarationScope _scope;
+        private readonly ConstantValue? _defaultValue;
```

Comment:
Let's move this down to `SynthesizedComplexParameterSymbol`, rather than putting it in the base, and override where appropriate (including for the caller info properties).